Doc/c-api/structures.rst

```diff
@@ -62,6 +62,38 @@ the definition of all other Python objects.
    See documentation of :c:type:`PyVarObject` above.
 
 
+.. c:function:: int Py_Is(const PyObject *x, const PyObject *y)
```

Comment:
Why not `c:macro::`? Just curious :)

Reply:
I like to pretend that it's a function. Moreover, the first version of my PR used a static inline function.

Reply:
They are now technically declared as functions **and** as static inline functions (using a macro overriding the symbol in the header file) ;-)